Configure fix for s390x, thanks to
Andreas Krebbel and Philipp Kern.

This patch is applied in get-orig-source
rule, prior to building the .orig.tar.gz file.



--- gmp-5.0.1+dfsg.orig/configure.in
+++ gmp-5.0.1+dfsg/configure.in
@@ -1095,8 +1095,13 @@
     ;;
 
 
-  # IBM s/370 and similar
-  [s3[6-9]0*-*-*])
+  # IBM S/390 64 bit
+  s390x-*-*)
+    gcc_cflags="-g -O2 $fomit_frame_pointer"
+    path="s390x"
+    ;;
+  # IBM S/390 32 bit
+  s390-*-*)
     gcc_cflags="$gcc_cflags $fomit_frame_pointer"
     path="s390"
     extra_functions="udiv_w_sdiv"
